net.xy.ykhwong.dosbox-daum.yml

```diff
@@ -0,0 +1,92 @@
+app-id: net.xy.ykhwong.dosbox-daum
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: dosbox
+build-options:
+  strip: true
+
+finish-args:         # flatpak permissions
+  - --device=all     # needed for OpenGL, gamepads and serial/parallel
+  - --share=ipc      # needed for X11
+  - --share=network  # Needed for IPX and serial over TCP/IP
+  - --socket=x11     # default to X11, as Gnome does not provide SSD
+  - --socket=pulseaudio
+  - --filesystem=home
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/share/aclocal'
+  - '/share/man'
+  - '*.la'
+  - '*.a'
+  - '/share/doc'
+
+modules:
+  - "shared-modules/glu/glu-9.json"
+  - "shared-modules/glew/glew.json"  
+  - name: SDL-1.2.15-hq
+    buildsystem: simple
+    build-commands:
+      - sh autogen.sh
+      - ./configure --prefix=/app/
+      - make #-e CXXFLAGS="-std=c++0x -stdlib=libstdc++"
+      - make install
+    sources:
+      - type: git
+        url: https://github.com/ebenbruyns/sdl1.2.15-hq.git
+        branch: main #  tag: release-1.2.15
```

Comment:
NEVER a branch.

Reply:
SDL 1.2.15 is in shared modules

https://github.com/flathub/shared-modules/blob/master/SDL/SDL-1.2.15.json

Reply:
That's a nice idea, and I fully agree with you, but this version of dosbox was created by special evil individuals that not only patched SDL 1.2 but also patched the gl.h file. The only way to get this to run on a modern system is flatpaks.